backend/api/tests/views/test_views_task_profiling.py

```diff
@@ -108,6 +108,17 @@ def test_task_profiling_create_success(self):
             {"count": len(expected_result), "next": None, "previous": None, "results": expected_result},
         )
 
+    def test_already_exist_task_profiling(self):
```

Comment:
Is this test failing without de serializer modification ?

Reply:
Yes, we began by adding the test